code/style.css

```diff
@@ -1,4 +1,300 @@
-html {
-  background: #0025ff;
-  color: #fff;
+body {
+  background-image: url('backgrounds/brushed_alu_dark.png');
+  background-attachment: fixed;
+  color: black;
+  margin: 0;
+  font-family: 'Barlow', sans-serif;
 }
+
+/* STYLING CREDIT TEXTS */
+.credit-text {
+  font-size: 12px;
+}
+
+/* HOVER EFFECTS ON CREDIT LINKS */
+.credit-text-links {
+  text-decoration: none;
+  color: black;
+}
+
+.credit-text-links:hover {
+  font-weight: bold;
+}
+
+/* HOVER EFFECTS ON "READ MORE" LINKS */
+.read-more-links {
+  color: black;
+  text-decoration: none;
+  position: relative;
+}
+
+.read-more-links:hover {
+  font-weight: bold;
+}
+
+.read-more-links::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  height: 1px;
+  background: black;
+
+  transform: scaleX(0);
+  transform-origin: right;
+  transition: transform 250ms ease-in;
+}
+
+.read-more-links:hover::after {
+  transform: scaleX(1);
+  transform-origin: left;
+}
+
+/*** LOGO AND HAMBURGER/NAVBAR ***/
+header {
+  background-color: rgba(255,255,255,0.5);
+  box-sizing: border-box; 
+  position: fixed;
+  top: 0;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  width: 100%;
+  padding: 12px 30px;
+  box-shadow: 5px 5px 5px rgba(129, 129, 129, 0.8);
+}
+
+.header-logo {
+  margin-right: auto;
+}
+
+.header-logo img {
+  width: 100px;
+  margin-right: 10px;
+}
+
+nav {
+  display: none;
+}
+
+/*** BIG NEWS SECTION ***/
+.big-news-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 90%;
+  margin: 150px auto;
```

Comment:
Maybe some less bottom margin to make top/bottom even (the top margin goes behind the sticky navbar so it looks like it's much smaller margin to the top)